### ISSUE TITLE
Clearer validation and instruction on NWC strings

### ISSUE THREAD

**Issue by huumn** [stackernews/stacker.news#1710]
Most people try to put the same string in for send/receive and our validation messages suck.

The rizful founder says everyone that tries to attach to SN is confused.

**Comment by ekzyis** [comment]
Interesting, they even have a [good guide](https://rizful.com/docs/connecting-apps#nostr-wallet-connect-send-and-receive).

This is somewhat related to #1497.

**Comment by huumn** [comment]
Yeah, most people using NWC are only using one string and don't understand why we need two. We probably need to go above an beyond explaining the reasoning and what permissions, exactly, each string requires.
